jobs/reset.js

```diff
@@ -82,40 +87,46 @@ module.exports = async function ({ repositoryFullName }) {
     endkey: `${repoDoc._id}:issue:\ufff0`,
     inclusive_end: true
   })
+  const openIssues = issues.rows.filter(issue => issue.doc.state !== 'closed')
+  log.info(`Started closing ${openIssues} open issues`)
```

Comment:
Don’t you want to log the length of the issues Array like before?

Reply:
And maybe put the issues themselves in the log payload.

Reply:
But much nicer with the filter 👍

Reply:
Of course :)